### PR TITLE
Post edit form UI tweaks

### PR DESCRIPTION
**Pull request by Will-Howard**
This is mainly the amber points on these designs: https://www.figma.com/file/DF9TLXQJuPrEJYEesS05J0/Social-preview?node-id=81%3A69&t=5rt8rtB5rn4yPQrI-0

Screenshot 2023-02-13 at 15 41 02
Screenshot 2023-02-13 at 15 41 16
Screenshot 2023-02-13 at 15 42 41
Screenshot 2023-02-13 at 15 42 48

Screenshot 2023-02-13 at 15 48 16

LessWrong:
Screenshot 2023-02-13 at 15 52 30

Done:
 - [X] Align the inner form sections
 - [X] Some tweaks to colors (mainly just to make them consistent, should be uncontroversial)
 - [X] Rename "Post topics" to "Set topics"
 - [X] Stop "This post may appear on the Frontpage" button from wrapping
 - [X] Update submit button styles
 - [X] Changing a few things to sentence case

Still left to do (may give up on):
 - [ ] There is a weird bug/feature where the "Save as draft" and "Submit" buttons reverse order below a certain screen size
 - [ ] Lighten the "Save as draft" and "post may appear on Frontpage" text in dark mode

### CODE REVIEW

**Comment by jpaddison3** [comment]
I have an Opinion about how to introduce the rounded corners that I'll need to send tomorrow.

**Comment by darkruby501** [comment]
Notes:
- We don't call them topics, we call them tags. So for us the section should be "Set tags", though I think "Apply Tags" would be better
- We don't want to change anything to sentence case
- We like the existing button styling (or at least, don't want to change the button styling here without considering button styling everywhere
- "This post may appear on the Frontpage" needs to wrap for us so there's room for "Request Feedback" button

File: packages/lesswrong/lib/collections/posts/formGroups.ts
Comment (darkruby501) [inline packages/lesswrong/lib/collections/posts/formGroups.ts:26]: Unfortunately don't want these changes

**Review by darkruby501 — DISMISSED** [review]
Hate to be a pain, but please ensure the changes are ForumGated before merging

File: packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx
Comment (agnesstenlund) [inline packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx:58]:
```suggestion
    padding: 6
```

Comment (agnesstenlund) [reply packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx:58]: Seeing this live, I think 6px radius would be better than the 8px I specified in the designs 👍

File: packages/lesswrong/lib/collections/posts/formGroups.ts
Comment (agnesstenlund) [inline packages/lesswrong/lib/collections/posts/formGroups.ts:88]:
```suggestion
    label: `Apply ${taggingNamePluralSetting.get()}`,
```

Agree with @darkruby501 on this one! Let's do `Apply` instead of` Set`

**Comment by agnesstenlund** [comment]
Thanks a lot for this @Will-Howard! 👑  looks so much more harmonious with the consistent alignment and colors.

The only thing I'd change: the button font size looks bigger than expected, would do 14px here instead of the specified 16px. But keep the same height of roughly 40px.

Intuitively `Apply` sounded more appropriate than `Set`, but seeing it in writing I think `Apply` draws too much attention from `topics`:

Screenshot 2023-02-14 at 16 22 21

So we should keep it as `Set topics` (but `Apply Tags` for LW)

**Comment by Will-Howard** [comment]
> * We don't call them topics, we call them tags. So for us the section should be "Set tags", though I think "Apply Tags" would be better

This is taken care of by the db setting 👍 
> * We don't want to change anything to sentence case

@agnesstenlund how strongly do you feel about sentence case vs not? Personally I care more about the cases being consistent than having everything as sentence case, so I would rather just keep them as they are (e.g. "Set Topics") to avoid the extra complexity of forum gating every label

> * We like the existing button styling (or at least, don't want to change the button styling here without considering button styling everywhere

Ok I will forum gate this

> * "This post may appear on the Frontpage" needs to wrap for us so there's room for "Request Feedback" button

It does still fit with the "Request feedback" when full width, and then the checkbox drops to the next line after a certain width (one of the screenshots has the request feedback button). I think this looks fine, but lmk what you think, I don't mind changing this one

**Comment by agnesstenlund** [comment]
actually feel pretty strongly in favour of using sentence case everywhere, so IMO worth forum gating (wrote a long blob about it [here](https://cea-core.slack.com/archives/CJY3ZKP62/p1675678999398949?thread_ts=1675543813.082219&cid=CJY3ZKP62))

**Comment by darkruby501** [comment]
> It does still fit with the "Request feedback" when full width, and then the checkbox drops to the next line after a certain width (one of the screenshots has the request feedback button). I think this looks fine, but lmk what you think, I don't mind changing this one

@Will-Howard cool, if you've checked it works, all good

**Comment by darkruby501** [comment]
> actually feel pretty strongly in favour of using sentence case everywhere, so IMO worth forum gating (wrote a long blob about it [here](https://cea-core.slack.com/archives/CJY3ZKP62/p1675678999398949?thread_ts=1675543813.082219&cid=CJY3ZKP62)) 🙏

Unfortunately links to CEA slack won't open for me (even when they're in a channel that's shared), can you copy here what you said? Thank you!

**Comment by agnesstenlund** [comment]
For sure, was in response to Raymond wanting to forum gate sentence case changes in Sarahs PR (https://github.com/ForumMagnum/ForumMagnum/pull/6633) (on why I want to have them for the EA Forum)

Screenshot 2023-02-15 at 10 28 45

**Comment by darkruby501** [comment]
> For sure, was in response to Raymond wanting to forum gate sentence case changes in Sarahs PR (#6633) (on why I want to have them for the EA Forum)
> 
> Screenshot 2023-02-15 at 10 28 45

Thanks @agnesstenlund I agree with most of your analysis. I also feel weirdly strongly about it. I think for me it's something like "It's not a sentence, though." Perhaps interesting, is there's something of a Rationalist writing habit (I think emulated from Eliezer) of capitalizing "noun phrases" in the middle of your prose, e.g. "the Man Who Shall Not Be Named came to visit yesterday", which perhaps makes me want things like "Advanced Options" to get the same treatment. But not the only thing. LessWrong's current style (and could change) is to be a bit more old-fashioned/formal and less friendly.

File: packages/lesswrong/components/posts/PostSubmit.tsx
Comment (s-cheng) [inline packages/lesswrong/components/posts/PostSubmit.tsx:14]: Prob should leave this at `16` for LW

File: packages/lesswrong/components/vulcan-forms/FormGroup.tsx
Comment (s-cheng) [inline packages/lesswrong/components/vulcan-forms/FormGroup.tsx:51]: @agnesstenlund I believe this border color change matches your designs, but to me the borders feel too faint now (at least on my screen), like it's hard to distinguish them. No worries if you want to keep it this color, but just thought I'd give my 2 cents.

Screen Shot 2023-02-15 at 6 12 00 PM

File: packages/lesswrong/components/vulcan-forms/FormGroup.tsx
Comment (s-cheng) [inline packages/lesswrong/components/vulcan-forms/FormGroup.tsx:54]: itshappening.gif

**Review by s-cheng — APPROVED** [review]
Nice thanks! I do think it would be nice to fix the "Save as draft" and "Submit" button order on mobile, but could just do it in a separate PR

**Comment by jpaddison3** [comment]
I'm terrible and still haven't commented. This will be rushed. I think we should hold off on the thing here that are part of the broad "style refactor", such as the buttons, and the border radius. (cc @agnesstenlund )

**Review by jpaddison3 — APPROVED** [review]
Ok, Will convinced me we should merge this anyway, and will have to refactor it later, but it's k.

**Comment by Will-Howard** [comment]
> Nice thanks! I do think it would be nice to fix the "Save as draft" and "Submit" button order on mobile, but could just do it in a separate PR

I just talked to JP about this and he thinks this was an intentional choice to make the submit button more natural to click on mobile. I'm not sure how I feel about this... but anyway I think this is a small enough issue that I'm just going to leave it (and not plan to fix it in a later PR), I expect that very few people submit posts from mobile anyway

**Pull request merged**
